### PR TITLE
Feature/31/create home page

### PR DESCRIPTION
**Pull request by PaulRomanov**
image

### CODE REVIEW

File: src/app/pages/home-page/home-page.component.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+.wrapper-description {
+  height: 15%;
+}
+
+.drop-box,
+.description {
+  height: 100%;
+  width: calc(100% - 234px);
```
Comment (vstrabko) [inline src/app/pages/home-page/home-page.component.scss:29]: I think things like width should also be moved into variable.

File: src/app/services/toast.service.spec.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
-import { TestBed } from '@angular/core/testing';
+// import { TestBed } from '@angular/core/testing';
```
Comment (vstrabko) [inline src/app/services/toast.service.spec.ts:1]: Why toaster service is commented?

File: src/app/services/toast.service.spec.ts [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+//   });
 
-  it('should be created', () => {
-    expect(service).toBeTruthy();
-  });
-});
+//   it('should be created', () => {
+//     expect(service).toBeTruthy();
+//   });
+// });
```
Comment (rastorc3v) [inline src/app/services/toast.service.spec.ts:16]: Why tests were commented?

File: src/app/pages/home-page/home-page.component.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<div class="wrapper-home-page">
+  <div class="wrapper-content">
+    <div class="wrapper-drop-box">
+      <select class="drop-box">
```
Comment (rastorc3v) [inline src/app/pages/home-page/home-page.component.html:4]: Don't you want to use material select?

File: src/app/pages/home-page/home-page.component.scss (hunk 1 above)
Comment (PaulRomanov) [reply src/app/pages/home-page/home-page.component.scss:29]: done

File: src/app/services/toast.service.spec.ts (hunk 2 above)
Comment (PaulRomanov) [reply src/app/services/toast.service.spec.ts:1]: fixed errors. I returned it as it was

File: src/app/pages/home-page/home-page.component.scss [hunk 5]
```diff
@@ -1,10 +1,16 @@
+$background-content:rgba(37, 51, 73, 0.85);
```
Comment (mhasilau) [inline src/app/pages/home-page/home-page.component.scss:1]: pls move colors to folder /src/app/shared/constats/colors.scss

Comment (PaulRomanov) [reply src/app/pages/home-page/home-page.component.scss]: done

**Review by Nicolai-boiko — APPROVED** [review]

**Pull request merged**